plugin-overlay/gradle.properties

```diff
@@ -2,5 +2,4 @@ POM_GROUP_ID=com.mapbox.plugin
 POM_ARTIFACT_ID=maps-overlay
 POM_ARTIFACT_TITLE=The map overlay module for the Mapbox Maps SDK for Android
 POM_DESCRIPTION=The map overlay module for the Mapbox Maps SDK for Android
-BINTRAY_PACKAGE_NAME=POM_GROUP_ID + ":" + POM_ARTIFACT_ID
```

Comment:
why did you remove it here? not for all modules?

Reply:
It's a leftover that's not present anywhere for a long time after we stopped publishing to Bintray

Reply:
And these 3 places were the only ones where this constant was still declared

Reply:
yeah, we can safely remove these fields as we are not publishing to bintray any more